specification/specs/x_pack/eql/EqlHitsEvent.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+class EqlHitsEvent<TEvent> {
+  _index?: IndexName
+  _id?: Id
+  _source?: TEvent
+  fields?: Dictionary<Field, UserDefinedValue[]>
```

Comment:
Same here.

Reply:
No everything is optional as well, look at this test: `xpack-Execute_EQL_sequence_with_filter_path_2_response.json.test-d.ts` e.g. where only `fields` is available.

Reply:
The reason is that the request is using `filter_path`, but in reality, those properties should be required.
This is something we should fix on the validator level.